.github/workflows/storybook-check.yml

```
@@ -0,0 +1,27 @@
+name: Check Storybook build
+
+on: pull_request
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    check:
+        runs-on: ubuntu-latest
+        if: ${{ github.repository == 'WordPress/gutenberg' }}
```

```suggestion
        if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
```

Not a blocker: Should we also run this CI on PRs submitted to forked repos? Some other CIs seem to do that. I think it's ok to go either way.

---

Good catch, I think I removed this part unintentionally 🙈